### PR TITLE
Pipe command line shall warn user on the JWT expiration

### PR DESCRIPTION
**Pull request by rodichenko**
Pipe command line shall warn user on the JWT expiration - done (#436)

### CODE REVIEW

**Review by sidoruka — COMMENTED** [review]
Does this functionality also considers `$API_TOKEN` value if `pipe` was configured via the environment variables?

**Comment by rodichenko** [comment]
> Does this functionality also considers `$API_TOKEN` value if `pipe` was configured via the environment variables?

Yes, jwt-token validation is implemented separately from the configuration itself; `pipe-cli` first reads `$API` & `$API_TOKEN` from the environment variables, then processes `configuration` file, and after that the validation occurs.

**Review by sidoruka — APPROVED** [review]

File: pipe-cli/src/config.py [hunk 1]
```diff
@@ -12,33 +12,133 @@
...
+def check_token(token, timezone, print_info=False):
+    if not token:
+        click.echo(click.style('No access token is provided', fg='red'), err=True)
+        return
+    try:
+        payload_decoded = token.split('.')[1]
+        # payload_decoded_correct_padding - is the same string with additional '=' symbols to make string length
+        # be dividable by 4:
+        payload_decoded_correct_padding = payload_decoded + b'=' * (4 - len(payload_decoded) % 4)
+        payload = json.loads(base64.b64decode(payload_decoded_correct_padding))
```
Comment (mzueva) [inline pipe-cli/src/config.py]: `jwt` library is already included into the project (take a look `src/utilitities/storage/common.py` -> `StorageOperations.get_user()`

**Review by mzueva — CHANGES_REQUESTED** [review]

File: pipe-cli/src/config.py [hunk 2]
```diff
@@ -12,33 +12,133 @@
...
+ALL_ERRORS = Exception
+
 
 class ConfigNotFoundError(Exception):
     def __init__(self):
         super(ConfigNotFoundError, self).__init__('Unable to locate configuration or it is incomplete. '
                                                   'You can configure pipe by running "pipe configure"')
 
 
+def check_token(token, timezone, print_info=False):
```
Comment (ekazachkova) [inline pipe-cli/src/config.py]: I would prefer not to make this file more complicated and move some methods into utilities as far as it possible

File: pipe-cli/src/config.py (hunk 1 above)
Comment (rodichenko) [reply pipe-cli/src/config.py]: @mzueva fixed via e74e38ae60622e3f862f2084f8e6a49145263748

File: pipe-cli/src/config.py (hunk 2 above)
Comment (rodichenko) [reply pipe-cli/src/config.py]: @ekazachkova I agree, I moved the validation logic to the `access_token_validation.py` utility (e74e38ae60622e3f862f2084f8e6a49145263748)

**Review by mzueva — APPROVED** [review]

**Review by ekazachkova — APPROVED** [review]

**Pull request merged**